### DISCUSSION TITLE
Dúvidas - Certificados de eventos passados (PyBR)

### DISCUSSION THREAD

**Discussion by jjpaulo2** [apyb/comunidade#120] [Pergunta]
Oi pessoal, tudo bem?

Eu tenho uma dúvida relacionada aos eventos passados da Python Brasil. Os certificados dos participantes de anos anteriores é mantido em algum lugar?

Eu descobri que não salvei o meu certificado da edição 2022, e o link que recebi por e-mail não funciona mais.

**Comment by naanadr** [comment]
@jjpaulo2 você tentou acessar a sua conta do eventbrite? Talvez lá você consiga acessar os eventos que você se inscreveu e os certificados 🤔

**Reply by jjpaulo2** [reply]
Pior que lá não aparece os certificados 😢

**Reply by naanadr** [reply]
Entrei no eventbrite, e também não encontrei onde que reenvia os formulários e nem onde que envia eles 😢
